en/docs/services/create-your-first-service.md

```diff
@@ -0,0 +1,181 @@
+# Create Your First Choreo Service
+
+To learn how to create a Choreo service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it and then observing its performance.
+
+## Step 1: Create the Choreo service
+
+To create the Choreo service, follow the procedure below: 
+
+1. Access the Choreo Development Console via the following URL.
+
+    `https://console.choreo.dev/`
+    
+    Sign in using either your Google or Github credentials.
+    
+2. In the **Services** page, click **Create**.
+
+3. Under **Create with Choreo**, enter `covid-stats` as the name of your Choreo service.
+
+4. In the **Configure API Trigger** form, select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
+
+    ![Resource Configuration](assets/img/services/configure-api-trigger.png)
+    
+    Then click **Save API**. Now you have saved the API trigger that starts your application.
+    
+5. In the next form that appears, do as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
+
+    1. Click **Connections**.
+
+    2. In the **New HTTP Connection** form, enter `https://api.covid19api.com` as the URL and select **Get** as the operation.
+    
+        ![New HTTP Connection](assets/img/services/new-http-connection.png)
+        
+        Then click **Save & Next**.
+        
+    3. In the **Select Payload Type** field, select **JSON**.
+    
+        ![Select Payload Type](assets/img/services/select-payload-type.png)
+        
+    4. Open the code view of the application by clicking the **<>** icon to the top right of the page.
+    
+        Update the `http:Response getResponse = <http:Response>check httpEndpoint->get("/");` line as `http:Response getResponse = <http:Response>check httpEndpoint->get("/total/country/united-states");`. This specifies that the GET statement in the HTTP connection invokes the public API to fetch the current number of active COVID-19 in United States.
+        
+    5. Click **Save & Done**. Now your API trigger is connected to the public API that fetches the active COVID-19 cases in the selected country.
+    
+6. To cast all the data retrieved via the API connected to the application before further processing, add a new custom statement as follows:
+
+    1. Click on the **+** icon below the JSON payload. 
+
+        ![Add New Statement](assets/img/services/add-new-statement.png)
+    
+    2. Then click **+** again to select the new statement you need to add.
```

Comment:
`+` is refereed with `+ icon` in the previous number one. But not with the `icon` word in this. Shall we follow a convention?

Reply:
Changed it and used the word "icon" in both places.

Reply:
There are more similar places. Check them all.

Reply:
Added "icon" in all places.